src/vcastTestInterface.ts

```diff
@@ -57,8 +57,28 @@ const path = require("path");
 
 export const vcastEnviroFile = "UNITDATA.VCD";
 
+// Creating a cache for the checksums so we don't constantly re-run the command
+interface checksumCacheType {
```

Comment:
Sonar doesn't like this: https://rules.sonarsource.com/typescript/type/Code%20Smell/RSPEC-101/

Reply:
what does it want Checksum *?

Reply:
Yeah